libs/designsystem/src/lib/scss/base/_variables.scss

```diff
@@ -98,3 +98,17 @@ $elevations: (
 ****************************************************************************/
 $alert-max-width: 359px;
 $compact-modal-max-width: 359px;
+
+/* Item
+****************************************************************************/
+$item-sizes: (
```

Comment:
Of very little importance....but.... it's not clear if it's height?.... size is a somewhat vague name?

Reply:
Renamed 👌